pkg/settlement/swap/chequebook/chequebook.go

```diff
@@ -135,3 +144,44 @@ func (s *service) WaitForDeposit(ctx context.Context, txHash common.Hash) error
 	}
 	return nil
 }
+
+// IssueCheque issues a new cheque
```

Comment:
add period end of sentence (godoc)

Reply:
fixed. added it to all the other sentences in chequebook package as well.